modules/javafx.graphics/src/main/java/com/sun/javafx/css/BitSet.java

```diff
@@ -235,7 +235,7 @@ public boolean containsAll(Collection<?> c) {
             return false;
         }
 
-        BitSet other = (BitSet)c;
+        BitSet<?> other = (BitSet<?>) c;
```

Comment:
It makes no difference, it could be `BitSet<String>` or `BitSet<? super T>` or anything really, because the type `T` is never used as part of this method.  `BitSet<?>` is the shortest form in that case.